store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/HBaseStore.java

```diff
@@ -180,6 +184,25 @@ protected SchemaOptimiser createSchemaOptimiser() {
     }
 
     @Override
+    public void validateSchemas() {
```

Comment:
So it looks like Accumulo, HBase and Parquet stores all have this same validation. Perhaps you could create 2 methods in Store with this logic.

validateConsistentVertex and validateConsistentGroupByProperties.

Then in the store implementations we can just call those 2 methods.